appveyor.yml

```diff
@@ -0,0 +1,11 @@
+environment:
+  matrix:
+    - TOXENV: py38
+
+build: off
+
+install:
+  - pip install tox==3.13.2
```

Comment:
Please use `python -m pip install --user git+https://github.com/blueyed/tox@master` here.

Reply:
Sure, though I am curious as to why. Why `python -m`? Why `--user`? What have you customized in your fork of tox?

Reply:
Could be still `pip`, but `python -m pip` often helps to use the right one.
As for `--user` that might be only useful on Travis / not even work here without the user location in PATH.
There are not many customizations, but e.g. it supports `--durations` (which you could add to the call then if you like, otherwise I'd might do it in general), the intitial motivation for having the fork.
As with the pytest one I'm gonna use it for "my" projects.  It would be nice if tox would have accepted the `--durations` flag (https://github.com/tox-dev/tox/issues/1450, https://github.com/tox-dev/tox/pull/1457), but well.